storage/redis/redis_scheduler_event_storage.go

```diff
@@ -41,13 +41,13 @@ func (es *RedisSchedulerEventStorage) PersistSchedulerEvent(event *models.Schedu
 	}
 
 	redisPipeline := es.redisClient.TxPipeline()
-	score := float64(event.CreatedAt.UnixNano())
+	score := float64(event.CreatedAt.UnixNano()	/ int64(time.Millisecond))
```

Comment:
Using it in nano broke the Redis somehow, or are you changing it to make it more "readable"?

Reply:
Making it more readable! 

```
redis-13098.internal.c13955.ap-neast-1-mz.ec2.cloud.rlrcp.com:13098> keys scheduler:*:events
1) "scheduler:suspects-room-blue:events"
redis-13098.internal.c13955.ap-neast-1-mz.ec2.cloud.rlrcp.com:13098> ZREVRANGEBYSCORE scheduler:suspects-room-blue:events +inf -inf WITHSCORES LIMIT 0 2
1) "{\"name\":\"AUTO_SCALE_START\",\"schedulerName\":\"suspects-room-blue\",\"createdAt\":\"2021-09-21T16:35:56.270816534Z\",\"metadata\":{\"amount\":9,\"type\":\"down\"}}"
2) "1.6322421562708165e+18"
3) "{\"name\":\"AUTO_SCALE_FINISHED\",\"schedulerName\":\"suspects-room-blue\",\"createdAt\":\"2021-09-21T16:30:08.330847465Z\",\"metadata\":{}}"
4) "1.6322418083308475e+18"
```

Is a mess to read.